packages/schematics/src/add/index.ts

```diff
@@ -135,18 +135,19 @@ export default function(options: ComponentOptions): Rule {
 
                 const scriptsSearch = scripts.join("|");
                 const stylesSearch = styles.join("|");
+                const pathPrefix = (json.apps) ? "../" : "";
     
-                if (stylesSearch.search("../node_modules/@clr/ui/clr-ui") < 0) {
-                    styles.push("../node_modules/@clr/ui/clr-ui.min.css");
+                if (stylesSearch.search("node_modules/@clr/ui/clr-ui") < 0) {
```

Comment:
why not just `indexOf` instead of `search`? Looks like we are not searching for a regular expression... 🤔